test/integration/content-sidebar/ContentSidebar.e2e.test.js

```diff
@@ -128,7 +128,7 @@ describe('ContentSidebar', () => {
 
     describe('activity feed comments', () => {
         const getDraftJSEditor = () => cy.getByTestId('bcs-CommentForm-body').find('[contenteditable]');
-        const getTooltip = () => cy.get('[role="tooltip"]');
```

Comment:
We generally want to avoid selecting on CSS classes for Cypress tests, except as a last resort. Can we either add a `data-testid` property to the tooltip or use some user-visible selector, such as the actual content of the tooltip?

Reply:
@jstoffan I just updated this PR with your comments